src/pages/docs/deployments/bulk-deployment-creation.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-09-13
+modDate: 2023-09-13
+title: Bulk deployment creation
+description: Octopus Deploy can use bulk deployment creation when deploying to multiple environments or tenants
+navOrder: 160
+---
+
+:::div{.hint}
+Bulk deployment creation will be available from Octopus Deploy **2023.4**
+:::
+
+When deploying a release to multiple environments or tenants through the Octopus Portal, the bulk deployment creation feature will immediately take you to a new server task which will create a deployment for each environment or tenant you are deploying to.
+
+In the bulk deployment creation server task, you can see the progress of deployments being created and any errors during creation.
+
+:::figure
+![](/docs/deployments/bulk-deployment-creation.png)
+:::
+
+You can also navigate away from the server task and it will continue creating deployments in the background.
+
+If you need to return to a bulk deployment creation server task, you can go to the **Tasks** page and filter down to **Bulk deployment creation** task types.
```

Comment:
Consider minor re-wording:

If you need to return to a bulk deployment creation server task, you can `navigate` to the **Tasks** page and use the task filter to select **Bulk deployment creation** task types.

Reply:
```suggestion
If you need to return to a bulk deployment creation server task, you can navigate to the **Tasks** page and use the task filter to select **Bulk deployment creation** task types.
```